robot/Cumulus/tests/browser/gift_entry/error_validation_rename_fields.robot

```diff
@@ -13,8 +13,9 @@ Suite Setup     Run keywords
 ...             Setup Test Data
 Suite Teardown  Run Keywords
 ...             Rename Object Field                     Account              custom_acc1_text     custom_acc_text
-...  AND        Query And Store Records To Delete       ${ns}DataImport__c   ${ns}NPSP_Data_Import_Batch__c=${BATCH1_Id}
-...  AND        Query And Store Records To Delete       ${ns}DataImport__c   ${ns}NPSP_Data_Import_Batch__c=${BATCH2_Id}
+...  AND        Query And Store Records To Delete       ${NS}DataImport__c   ${NS}NPSP_Data_Import_Batch__c=${BATCH1_Id}
+...  AND        Query And Store Records To Delete       ${NS}DataImport__c   ${NS}NPSP_Data_Import_Batch__c=${BATCH2_Id}
+...  AND        Delete Session Records
```

Comment:
Why are you adding this specifically again when line 19 is already deleting session records?

Reply:
Left this in by accident. Removed it.